File: Python/1281.py
a, b = map(int, input().split())
sum = 0

for i in range(a, b + 1):
    if i==a and i%2!=0:
        print(str(i), end='')
        sum += i

    elif i==a and i%2==0:
        print("-"+str(i), end='')
        sum -= i

    elif i%2!=0:
        print("+"+str(i), end='')
        sum += i

    elif i%2==0:
        print("-"+str(i), end='')
        sum -= i

print("="+str(sum))

File: C/1281.c
#include <stdio.h>

int main(){
	int a, b, sum = 0;
	scanf("%d %d", &a, &b);
	
	for(int i = a; i <= b; i++){
		if(i==a && i%2!=0){
			printf("%d", i);
			sum += i;
			
		} else if(i==a && i%2==0){
			printf("-%d", i);
			sum -= i;
			
		} else if(i%2!=0){
			printf("+%d", i);
			sum += i;
			
		} else if(i%2==0){
			printf("-%d", i);
			sum -= i;
			
		}
	}
	printf("=%d", sum);
	return 0;
}